elastic/handlers.go

```diff
@@ -1,29 +1,32 @@
 package elastic
 
 import (
-	"bytes"
 	"encoding/json"
-	"log"
+	"fmt"
+	"io/ioutil"
 	"net/http"
 )
 
-func tHandler(w http.ResponseWriter, r *http.Request) {
-	buf := new(bytes.Buffer)
-	buf.ReadFrom(r.Body)
-	bodyStr := buf.String()
-	body := []byte(bodyStr)
-	encoded := EncodeQuery(body)
-	b, err := json.Marshal(encoded)
+func translateHandler(w http.ResponseWriter, r *http.Request) {
+	b, err := ioutil.ReadAll(r.Body)
+	defer r.Body.Close()
 	if err != nil {
-		log.Panic(err)
+		return
 	}
+	var t Term
+	json.Unmarshal(b, &t)
```

Comment:
> `w.WriteHeader(http.StatusUnprocessableEntity)`
This constant looks to be a pretty recent addition to Go, my compiler doesn't recognize it. Still use?